File: format.cpp
#include<iostream>
#include<ctime>
#define debug
#define FILENAME "00000_format/format_in.txt"
#define FILE_CIN
#define elapsedseconds(start) (clock()-start)/(double)CLOCKS_PER_SEC
using namespace std;
//보통 연산량 1억에 1초이지만
//단순 수식계산, 메모리 접근, 자료구조에 따라 연산량은 더 늘어나거나 줄어들 수 있다.
int main(){
#ifdef FILE_CIN
	freopen(FILENAME,"r",stdin);
	int tmp;
	cin>>tmp;
	for(int z=0;z<tmp;z++){
#endif
        clock_t start=clock();
        cout<<elapsedseconds(start)<<endl;
#ifdef FILE_CIN
	}
#endif
	return 0;
}

File: format.java
//자바 파일이름은 반드시 Main.java
//제출시 패키지 커멘트 아웃
package format;
import java.util.Scanner;
import java.io.FileInputStream;
public class Main{
    private static final boolean file_cin=true;
    private static final String filename="0000_format/format_in.txt";
    public static void main(String[] args){
        if(file_cin){
            FileInputStream instream=null;
            try{
                instream = new FileInputStream(filename);
                System.setIn(instream);
            }
            catch(Exception e){
                e.printStackTrace();
            }
        }
        Scanner in = new Scanner(System.in);
        int tmp;
        if(file_cin)
            tmp=in.nextInt();
	    for(int z=0;z<tmp;z++){
        }
    }
}